--- am-utils-6.0a16/include/am_defs.h.glibc21	Thu Jan 21 11:15:18 1999
+++ am-utils-6.0a16/include/am_defs.h	Thu Jan 21 11:23:54 1999
@@ -308,7 +308,7 @@
  * like Linux, it also defines "struct datum".
  */
 #ifdef HAVE_NDBM_H
-# include <ndbm.h>
+# include <db1/ndbm.h>
 # ifndef DATUM
 /* ensure that struct datum is not included again from <rpcsvc/yp_prot.h> */
 #  define DATUM
--- am-utils-6.0a16/libamu/xutil.c.glibc21	Thu Jan 21 11:17:44 1999
+++ am-utils-6.0a16/libamu/xutil.c	Thu Jan 21 11:19:14 1999
@@ -48,7 +48,7 @@
 #include <am_defs.h>
 #include <amu.h>
 
-FILE *logfp = stderr;		/* Log errors to stderr initially */
+FILE *logfp = NULL;		/* Log errors to stderr initially */
 
 #ifdef HAVE_SYSLOG
 int syslogging;
